### ISSUE TITLE
Zone scope definition not resolved correctly in PrimePlotter

### ISSUE THREAD

**Issue by waltersma** [ansys/pyprimemesh#830]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Incorrect face zonelets displayed in [Example](https://github.com/ansys/pyprimemesh/blob/main/examples/gallery/00_lucid_file_IO.py).

Following scope definition for zones is not resolving to the intended faces:

```
for zone in part.get_face_zones():
    print(model.get_zone_name(zone))
    scope = prime.ScopeDefinition(
        model,
        evaluation_type=prime.ScopeEvaluationType.ZONES,
        zone_expression=model.get_zone_name(zone),
    )
    display = PrimePlotter()
    display.add_scope(model, scope)
    display.show()
```

### 📝 Steps to reproduce

Run the linked example case until the final display code section quoted. 

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

24.2.0

### 🐍 Which Python version are you using?

3.10

### 📦 Installed packages

```shell
Running CI/CD.
```

**Comment by waltersma** [comment]
Summary of issues noted:

- [00_lucid_file_IO.py](https://github.com/ansys/pyprimemesh/blob/main/examples/gallery/00_lucid_file_IO.py) - showing incorrect surfaces from zones
- [03_lucid_pipe_tee.py](https://github.com/ansys/pyprimemesh/blob/main/examples/gallery/03_lucid_pipe_tee.py) - not showing poly mesh in final image
- [05_pcb_stacker.py](https://github.com/ansys/pyprimemesh/blob/main/examples/gallery/05_pcb_stacker.py) - not showing surface mesh on second plane display
- [08_lucid_generic_f1_rear_wing.py](https://github.com/ansys/pyprimemesh/blob/main/examples/gallery/08_lucid_generic_f1_rear_wing.py) - not showing poly mesh in final image

**Comment by AlejandroFernandezLuces** [comment]
Closed by #851  and #861
